great-idea/css/index.css

```diff
@@ -63,4 +63,275 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+body {
+	width: 876px;
+	margin: 0 auto;
+  }
+
+h1 {
+	  font-size: 60px;
+  }
+
+.navigation {
+	display: flex;
+	width: 876px;
+	justify-content: space-around;
+	color: grey;
+}
+
+.links {
+	display: flex;
+
+	align-items: flex-start;
+	word-spacing: 50px;
+}
+
+a {
+	padding-right: 20px;
+}
+
+.logo {
+	display: flex;
+	justify-content: flex-end;
+
+}
+
+/*
+.navigation {
+	display: inline-block;
+	position: fixed;
+
+	margin-bottom: 15px;
+	background: white;
+}
+
+.links {
+	color: grey;
+	display: inline-block;
+	width: 590px;
+	height: 50px;
+	padding: 20px 0 20px 0;
+	margin: 15px auto;
+
+	word-spacing: 50px;
+}
+
+.logo{
+	display: inline-block;
+	right: 0;
+	width: 290px;
+	height: 50px;
+	margin: 15px auto;
+	padding: 0 0 0 100px;
+
+}
+*/
+
+.head {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	margin: 40px auto;
+
+}
+
+.title-button {
+	display: flex;
+	flex-direction: column;
+	text-align: center;
+	align-items: center;
+	margin-left: 100px;
+}
+
+.circle-logo {
+	display: flex;
+	justify-content: flex-end;
+}
+
+/*
+
+
+.title-button {
+	width: 445px;
+	height: 245px;
+	display: inline-block;
+	margin: 0 auto 0; 
+	text-align: center;
+	padding-top: 70px;
+	line-height: normal;
+	vertical-align: middle;
+
+}
+
+.title{
+
+}
+
+.button {
+
+}
+
+
+.circle-logo {
+	width: 445px;
+	height: 445px;
+	display: inline-block;
+	margin: 120px auto 0;
+	vertical-align: middle;
+
+}
+*/
+
+.top-content {
+	display: flex;
+	justify-content: space-around;
+	margin: 20px auto;
+}
+
+/*
+.features {
+	width: 445px;
+	height: 100px;
+	margin: 20px auto;
+	display: inline-block;
+}
+
+.about {
+	width: 445px;
+	height: 100px;
+	margin: 20px auto;
+	display: inline-block;
+}
+*/
+
+.bottom-content {
+	display: flex;
+	justify-content: space-between;
+	margin: 20px auto;
+}
+
+.services {
+	padding-right: 5px;
+}
+
+
+.vision{
+	padding-left: 5px;
+}
+
+/*
+.services {
+	width: 291px;
+	height: 160px;
+	margin: 20px auto;
+	display: inline-block;
+}
+
+.product {
+	width: 291px;
+	height: 160px;
+	margin: 20px auto;
+	display: inline-block;
+}
+
+.vision {
+	width: 291px;
+	height: 160px;
+	margin: 20px auto;
+	display: inline-block;
+}
+*/
+.contact {
+	line-height: 2;
+}
+
+footer {
+	padding: 20px 0;
+	text-align: center;
+}
+
+
+
+
+
+
+/* services.html CSS  below  */
+
+.features {
+	margin-top: 20px;
+}
+
+button {
+	margin-top: 5px;
+	padding: 5px 20px;
+	border: 1px solid black;
+}
+
+.middle-content {
+	display: flex;
+	align-content: space-evenly;
+	flex-wrap: wrap;
+
+}
+
+.digital-design {
```

Comment:
Whenever you have repeating code like this it's good practice to make it more DRY by having them all be the same class name.